src/cpp/rtps/participant/RTPSParticipantImpl.h

```diff
@@ -126,7 +126,24 @@ class RTPSParticipantImpl
         * @param part
         * @param plisten
         */
-    RTPSParticipantImpl(const RTPSParticipantAttributes &param, const GuidPrefix_t& guidP, RTPSParticipant* part,
+    RTPSParticipantImpl(
```

Comment:
Wrong indentation on comments and parameters

Reply:
ok